heron/spi/src/java/com/twitter/heron/spi/utils/SchedulerUtils.java

```diff
@@ -212,7 +212,7 @@ private SchedulerUtils() {
     commands.add(Context.systemConfigSandboxFile(config));
     commands.add(Runtime.componentRamMap(runtime));
     commands.add(SchedulerUtils.encodeJavaOpts(TopologyUtils.getComponentJvmOptions(topology)));
-    commands.add(Context.topologyPackageType(config));
+    commands.add(Context.topologyPackageType(config).toString());
```

Comment:
better to use `.name()` than toString. Also check on the cases, think both return caps.